7-standards-assessment.adoc

```diff
@@ -903,16 +903,68 @@ In addition, the other parameters of the GetScene request (beside `request`,
 
-It is planned by HFT to implement an experimental 3D Portrayal Service scene
-resource following this approach based on the Testbed 13 showcase using the
-3D CityGML model of New York described <<uc-3d,here>>. The request
+
+Steinbeis Transfer Center at HFT Stuttgart (Steinbeis) has implemented an experimental 3D Portrayal Service scene
+resource following the approach based on the Testbed 13 showcase using the
+3D CityGML model of New York described http://docs.opengeospatial.org/per/17-046.html[here]. The implementations is based on openAPI and the REST API can be tested http://steinbeis-3dps.eu:8080/tb14/API/index.html[http://steinbeis-3dps.eu:8080/tb14/API/index.html].  
+
+Alternatively, to use the REST API call in your application make the following request by defining the correct **bbox** coordinates
+
 
 ----
-../3D_CityModel_manhattan/collections/buildings/scene?bbox=-74.0064,40.7178,-73.9739,40.7507
+ http://steinbeis-3dps.eu:8080/tb14/wfs3/3D_CityModel_manhattan/collections/buildings/scene?format=application/json&bbox=-73.99562241015136,40.72227595290325,-73.9799164993194,40.749273379993284
 ----
 
-will deliver a 3D building model (as one layer of the 3D city model (3D-DLM))
-of Manhattan using 3D Tiles. It can be rendered in the Cesium globe.
+The aforementioned call will deliver a 3D building model (as one layer of the 3D city model (3D-DLM))
+of Manhattan using 3D Tiles. It can be rendered in the Cesium globe (See Fig 1). A prototype implemetation can be accessed (http://81.169.187.7:8081/3DPS_App/)
+
+
+[#img-App]
+.Cesium Globe 3D Tiles Manhattan 
+image::images/3dps6.png[3D]
+
+The current OpenAPI implementation supports two functionalities **Data Layer** and **3D Scene** (See Fig. 2). 
+
+[#img-App2]
+.OpenAPI 
+image::images/3dps1.png[]
+
+The **data Layer** function is used to retrieve the name of the available data layers. The current implementation supports one layer only:  _“3D_CityModel_manhattan”_ . However, in case the service supports more then one data layer it will show a list of supported data layers.
+
+----
+{“datalayer ”: “3D_CityModel_manhattan,3D_CityModel_geneva,3D_CityModel_stuttgart"}
+----
+
+To test the REST API call, first clicking on the _Try it out_  button and then _execute_ button.
+
+[#img-App3]
+.OpenAPI Data Layer
+image::images/3dps2.png[]
+
+Since this call does not take any input you will see the output as shown in Fig 3. In addition, it also shows the curl request with all the necessary query parameters.
+
+The **3D Scene** function on the other hand takes three input. The first is the path parameter(_layer_) and the remaining two are query parameters(_format_,_bbox_) see Fig 4
+
+[#img-App4]
+.OpenAPI 3D Scene
+image::images/3dps3.png[]
+
+A Successful execution will provide the a tilesetlink (see example below)
+
+----
+tilesetLink :http://tb13.igd.fraunhofer.de:8083/3dps/datahoster/fc38b549-4989-4703-bb27-f5b4fd33fd90/manhattan/tileset.json
```

Comment:
This should be a `link` (https://github.com/opengeospatial/WFS_FES/blob/master/core/openapi/schemas/link.yaml). Again, perhaps just add a note about the simplified approach taken in the prototype.